tests/utils/utils.go

```diff
@@ -77,25 +79,38 @@ func DeployERC20TokenHome(
 	teleporterManager common.Address,
 	tokenAddress common.Address,
 	tokenHomeDecimals uint8,
-) (common.Address, *erc20tokenhome.ERC20TokenHome) {
+) (common.Address, *proxyadmin.ProxyAdmin, *erc20tokenhome.ERC20TokenHome) {
 	opts, err := bind.NewKeyedTransactorWithChainID(
 		senderKey,
 		subnet.EVMChainID,
 	)
 	Expect(err).Should(BeNil())
-	address, tx, erc20TokenHome, err := erc20tokenhome.DeployERC20TokenHome(
+	implAddress, tx, _, err := erc20tokenhome.DeployERC20TokenHome(
 		opts,
 		subnet.RPCClient,
+	)
+	Expect(err).Should(BeNil())
+	teleporterUtils.WaitForTransactionSuccess(ctx, subnet, tx.Hash())
+
+	proxyAddress, proxyAdmin, erc20TokenHome := DeployTransparentUpgradeableProxy(
```

Comment:
Don't think it effects the test run times too significantly, but should we consider having a default non-upgradeable deployment here as well as an upgradeable deployment helper?

I guess either way it's 2 transactions right now, so likely doesn't make a difference.

Reply:
in the non-upgradeable extension by default they'll use the non-upgradeable extesnsions